docs/releases/1.10.txt

```diff
@@ -53,6 +53,10 @@ Minor features
   to prevent an issue where Safari caches redirects and prevents a user from
   being able to log out.
 
+* The :func:`~django.contrib.auth.views.login` view has a new parameter
```

Comment:
I try to avoid passive voice: The new ``redirect_authenticated_user`` parameter for the login view allows automatically...